src/remote-config.ts

```diff
@@ -31,40 +31,36 @@ export class RemoteConfigLoader {
     }
 
     load(): void {
-        // Call decide to get what features are enabled and other settings.
-        // As a reminder, if the /decide endpoint is disabled, feature flags, toolbar, session recording, autocapture,
-        // and compression will not be available.
-
-        if (!this.instance.config.__preview_remote_config) {
-            return
-        }
-
-        // Attempt 1 - use the pre-loaded config if it came as part of the token-specific array.js
-        if (assignableWindow._POSTHOG_CONFIG) {
-            logger.info('Using preloaded remote config', assignableWindow._POSTHOG_CONFIG)
-            this.onRemoteConfig(assignableWindow._POSTHOG_CONFIG)
-            return
-        }
-
-        if (this.instance.config.advanced_disable_decide) {
-            // This setting is essentially saying "dont call external APIs" hence we respect it here
-            logger.warn('Remote config is disabled. Falling back to local config.')
-            return
-        }
+        try {
```

Comment:
diff is just the try catch and moving the "preview" check into the callback function